blocks/quote/quote.js

```diff
@@ -0,0 +1,38 @@
+import { createOptimizedPicture, decorateIcons } from '../../scripts/aem.js';
+import { span } from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  /* change to ul, li */
+  const ul = document.createElement('ul');
+  [...block.children].forEach((row) => {
+    const li = document.createElement('li');
+    while (row.firstElementChild) li.append(row.firstElementChild);
+    [...li.children].forEach((divElement) => {
+      if (
+        divElement.children.length === 1
+        && divElement.querySelector('picture')
+      ) {
+        divElement.className = 'cards-card-image';
+      } else divElement.className = 'cards-card-body';
```

Comment:
why did you choose cards-card-body and cards-card-image as class names?

Reply:
didn't think to change them after copying that part of the code

I will change to quote-block-image and quote-block-body